apps/els_lsp/src/els_dt_document.erl

```diff
@@ -168,3 +172,27 @@ get_element_at_pos(Item, Line, Column) ->
 -spec uri(item()) -> uri().
 uri(#{ uri := Uri }) ->
   Uri.
+
+-spec functions_at_pos(item(), non_neg_integer(), non_neg_integer()) -> [poi()].
+functions_at_pos(Item, Line, Column) ->
+  POIs = get_element_at_pos(Item, Line, Column),
+  [POI || #{kind := 'function'} = POI <- POIs].
+
+-spec applications_at_pos(item(), non_neg_integer(), non_neg_integer()) ->
+        [poi()].
+applications_at_pos(Item, Line, Column) ->
+  POIs = get_element_at_pos(Item, Line, Column),
+  [POI || #{kind := 'application'} = POI <- POIs].
+
+-spec wrapping_functions(item(), non_neg_integer(), non_neg_integer()) ->
+        [poi()].
+wrapping_functions(Document, Line, Column) ->
```

Comment:
Is this expected to return more than one function?

How is the behavior with funs?

Reply:
Good eye :)
It will be always be  a `[]` or a list with exactly one element. Returning a list allows to avoid `case`ing around in the calling code (if the list is empty, `[]` will be sent back to the client).

Reply:
This is not taking funs into account at this stage.